swagger.yaml

```diff
@@ -2243,7 +2324,10 @@ definitions:
         description: Json formatted string containing the user overrides for the install-config.yaml file
         example: '{"networking":{"networkType": "OVN-Kubernetes"},"fips":true}'
         x-go-custom-tag: gorm:"type:varchar(2048)"
-
+      ignition_config_overrides:
+        type: string
+        description: Json formatted string containing the user overrides for the initial ignition config
+        example: '{"storage": {"files": [{"path": "/tmp/example", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}'
```

Comment:
What will happen in case of version mismatch?
From the [code](https://github.com/openshift/assisted-service/pull/310/files#diff-97c2fe56dacc4fee9ab38e4dbd281ab8R261) it seems we only support 3.1 
Perhaps the swagger should reflect that.

Reply:
The [code here](https://github.com/openshift/assisted-service/pull/310/files#diff-97c2fe56dacc4fee9ab38e4dbd281ab8R379-R382) should return a 400, but I'll add a test case for this and update this example.